lacci/lib/shoes/unforced_dependencies/bloops.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Bloops
+  # SAW, SINE, SQUARE and friends
+  def const_missing(name)
+    begin
+      require "bloops"
+      Bloops.const_get(name)
+    rescue LoadError
+      raise "\n\n== You need to install Bloops! ==\n\n== If you want to jam to sweet tunes ==\n\n"
+    end
+  end
+
+  def initialize
+    begin
+      require "bloops"
```

Comment:
@noahgibbs I feel like I'm desperately close. There may be some more failures on the next line of course. but this require is still giving me a failure after I've gotten bloops manually installed onto my machine.

Using command

`SCARPE_HTML_RENDERER=tiranti ./exe/scarpe examples/bloopsaphone/working/b1.rb --dev --debug`

Still getting 

`/Users/schwad/OSS/scarpe/lacci/lib/shoes/unforced_dependencies/bloops.rb:18:in `require': cannot load such file -- bloops (LoadError)`

But when I require bloops in `irb` it behaves. 

Any tips or thoughts? Tried to slog it out myself but figured I'd post where I'm at for ideas 😅

Reply:
But problem here is that we're starting Bundler in exe/scarpe. And Bundler prevents using any undeclared gems. So that's almost certainly your problem here.

Reply:
I'm not quite sure what to suggest. This is actually a fairly serious known Bundler problem - you'll often see the same kind of thing with wanting to use a debug gem (e.g. irb, pry) in an app that doesn't have them declared in the Gemfile. You can kinda-sorta do it with gem groups, but then the person wanting to use Bloops has to declare the groups to use somehow based on whether they have bloops :-/

Reply:
One thing you could maybe do, but I think it would be really annoying, would be to have a build step for Bloops as a shared library and build it when needed. Then you could declare the Bloops gem whether it built successfully or not. Basically, do this same thing but *inside* bloops. But then you have to modify bloops *and* you have to do the build step on-demand at runtime. So, not a trivial thing.

Reply:
If the problem with Bloops happened at runtime you could do some kind of error-tolerant autoload. It's a lot harder with it being at compile time :-(